wordpress-mobile/WordPress-iOS
# Stats Insights: use WordPressFlux for data management

Ref #10374 (part 2 of 3)

This adds a `StatsInsightsStore` to handle retrieving Insights data and updating the view via the `SiteStatsInsightsViewModel`. At this time, the only data returned is the Latest Post Summary.

To test:
Sanity checks on these three states to make sure all works as expected.
- Post has data.
  - Verify `View more` shows the alert.
- Post has no data.
  - Verify `Share Post` shows sharing options.
- Site has no posts. 
  - Verify `Create Post` shows post creation.
  - After creating a post, verify the Latest Post Summary is updated (it may take a couple of seconds).

NOTE: @jklausa you may want to review https://github.com/wordpress-mobile/WordPress-iOS/pull/10442 first, as this PR builds on that one and the progression may be clearer.
